include/libtorrent/kademlia/observer.hpp

```diff
@@ -38,6 +38,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #include <libtorrent/time.hpp>
 #include <libtorrent/address.hpp>
+#include <libtorrent/bdecode.hpp>
```

Comment:
please forward declare ``bdecode_node`` instead of including this header. (you can include the header in the .cpp file)